src/code/providers/lara-provider.coffee

```diff
@@ -279,8 +279,7 @@ class LaraProvider extends ProviderInterface
     callback "Cannot open the specified document"
 
   getOpenSavedParams: (metadata) ->
-    @encodeParams if metadata.providerData.url \
-                    then { url: metadata.providerData.url } \
-                    else metadata.providerData
+    # if loaded from lara then reuse the params so we don't leak the keys
+    @encodeParams @laraParams or metadata.providerData
```

Comment:
My assumption here was that after the initial open the run state would be updated so that other fields of the `launchWithLara` params would not be required (or desired). At the time, I was specifically thinking of the `source` parameter, which is clearly not needed after the initial open. Is there a scenario where the access keys need to be in the URL because they're not saved in the run state? Wouldn't that mean that anyone that got hold of the URL could potentially open and/or edit the document, which was what we were trying to get away from by storing the access keys in the run state?